src/main/java/persistence/sql/ddl/TableInfo.java

```diff
@@ -0,0 +1,27 @@
+package persistence.sql.ddl;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+public class TableInfo {
+    Class<?> entity;
```

Comment:
```suggestion
public class TableInfo {
    private final Class<?> entity;
```

접근 제어자가 빠진 것 같아요 :)
또한, 불변을 지키면 어떤 효과가 있을까요?